pkg/core/file_ref.go

```diff
@@ -21,7 +21,11 @@ func (r *FileRef) Path() string {
 }
 
 func (r *FileRef) WriteTo(w io.Writer) (int64, error) {
```

Comment:
The issue/PR description mention wanting to look at files that aren't in the project directory, but this change rather seems to make it possible to write to any directory on the local machine.

Reply:
well we wouldnt write to any directory on the local machine becuase we join the path with the execution unit or static units compiled path output. Its already possible, the real question is do we want to allow klotho to look at files outside of the scope passed in through the CLI and that i dont know the answer to